packages/x-components/cypress.config.ts

```diff
@@ -0,0 +1,45 @@
+import { addCucumberPreprocessorPlugin } from '@badeball/cypress-cucumber-preprocessor';
+import { createEsbuildPlugin } from '@badeball/cypress-cucumber-preprocessor/esbuild';
+// @ts-expect-error TS definitions are wrong
+import createBundler from '@bahmutov/cypress-esbuild-preprocessor';
+import { defineConfig } from 'cypress';
+
+export default defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:8080',
+    defaultCommandTimeout: 7000,
+    viewportHeight: 1080,
+    viewportWidth: 1920,
+    screenshotOnRunFailure: false,
+    video: false,
+    supportFile: 'tests/support/index.ts',
+    fixturesFolder: 'tests/e2e/fixtures',
+    screenshotsFolder: 'tests/e2e/screenshots',
+    retries: {
+      openMode: 0,
+      runMode: 1
+    },
+    async setupNodeEvents(on, config) {
+      await addCucumberPreprocessorPlugin(on, config);
+      on(
+        'file:preprocessor',
+        createBundler({
+          plugins: [createEsbuildPlugin(config)]
+        })
+      );
+      return config;
+    },
+    specPattern: 'tests/e2e/**/*.feature'
+  },
+  component: {
```

Comment:
I was watching the migration video that they have, and they suggest renaming all the tests to `*.cy.ts`, but as we are using the specPattern, I'm not sure if this is necessary.

<img width="895" alt="image" src="https://user-images.githubusercontent.com/6597815/207560630-893870cd-df33-4f67-8640-161fced190ee.png">

Also, they renamed the support file to be `e2e.js` or `component.js`. I suppose that all these changes makes sense if you are not setting the configuration as we do.

<img width="922" alt="image" src="https://user-images.githubusercontent.com/6597815/207560946-1ef92e6a-dc34-4401-a2a8-2f39e0aece3c.png">

Reply:
I would leave it as it is for now. Enough noise in this PR to start renaming more files. We can do that in the future if we need it.
About the support file, we are sharing the same support file from e2e and component. This right now only makes sense to have the same `getByDataTest` command and avoid creating more support and shared files.